code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -123,6 +233,19 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	dangerous = TRUE
 	fusion_power = 5
 	rarity = 300
+	cond_rate = 0.05
+	cond_temp_max = BP_TRITIUM
+	cond_temp_min = TCMB
+	cond_type = /datum/reagent/hydrogen
+	cond_event = TRUE
+	cond_heat = 1390
+
+/datum/gas/tritium/on_condense(datum/gas_mixture/holder, cond_moles, cond_temp, atom/location, datum/reagents/target, list/cond_rates)
+	. = ..()
+	if(cond_moles < TRITIUM_MINIMUM_RADIATION_ENERGY)
+		return
+	if(prob(50))
+		radiation_pulse(location, cond_moles / TRITIUM_MINIMUM_RADIATION_ENERGY)
```

Comment:
Gas types are not objects, they're templates. Why is this being done here rather then in some condensation specific bit of code?

Reply:
Because if I chucked it into condensation code it would be more difficult to expand on later.